docs/source/quick start.rst

```diff
@@ -82,66 +82,66 @@ Warning: The first of the following commands is not well tested. If you run into
 
 .. code-block:: sh
    :emphasize-lines: 1,2
-   
+
    brew install pygobject3 gtk+3 cairo py3cairo pkg-config
    python3 -m pip install --upgrade https://github.com/CoffeeStraw/PyonFX/zipball/master
 
 Warning: If you experience output not rendered correctly, you might need to change the PangoCairo backend to fontconfig.
 
 .. code-block:: sh
    :emphasize-lines: 1
-   
+
    PANGOCAIRO_BACKEND=fc python3 namefile.py
 
 
 Installation - Extra Step
 +++++++++++++++++++++++++
 
-This step is not needed to start working with the library, but personally I consider Aegisub quite old and heavy, so I needed a more comfortable way to work.
+This step is not mandatory to start working with the library, but I personally consider Aegisub to be quite old and heavy, so I needed a more comfortable work setup.
 
-That's why PyonFX integrates an additional way to reproduce your works in softsub faster after each generation, using the `MPV player <https://mpv.io/>`_. Installing it should be enough to make everything work if your're NOT on Windows.
+That's why PyonFX integrates an additional way to reproduce your works in softsub faster after each generation, using the `MPV player <https://mpv.io/>`_. Installing it should be enough to make everything work if you're **not** on Windows.
 
-If you're on Windows, all you need to do once you have installed it (check the website for that), is to add it to the PATH, so that the library will be able to utilize it. There are several guide for that, `here you can find one <https://www.architectryan.com/2018/03/17/add-to-the-path-on-windows-10/>`_.
+If you're on Windows, you will need to add it to PATH after downloading it so the library will be able to utilize it. There are several guides for that, `like this one <https://www.architectryan.com/2018/03/17/add-to-the-path-on-windows-10/>`_.
 
 You need to add the folder that contains the .exe of mpv, generally C:\\Program Files\\mpv.
 
 
-Starting
+Starting out
 ++++++++
 
-You may want to check if everything is working nicely now. For that, I suggest you to try running some of the examples in the `GitHub official repository of the project <https://github.com/CoffeeStraw/PyonFX/tree/master/examples>`_.
+Before starting, you may want to make sure everything functions as intended. I suggest you to try running some of the examples in the `official GitHub repository of the project <https://github.com/CoffeeStraw/PyonFX/tree/master/examples>`_.
 
-To run a script in python, all you need to do is run the following command:
+To run a script in python, run the following command:
 
 .. code-block:: sh
    :emphasize-lines: 1
 
    python namefile.py
```

Comment:
See the previous comment: It's probably safer to write Python3 here as well.

Reply:
I would keep the suggestion to try both (with priority to "python" as the majority of people use Windows).